src/hooks/use-disable-main-button.hook.ts

```diff
@@ -0,0 +1,20 @@
+import {useEffect} from 'react';
+
+import {DEFAULT_HINT_COLOR} from '../globals';
```

Comment:
as `DEFAULT_HINT_COLOR` is used only in this hook - now it make no sense to move it to `globals`